ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -1111,6 +1422,7 @@ NetworksForm.propTypes = {
   submitCallback: PropTypes.func,
   restrictHeight: PropTypes.bool,
   setActiveOnSubmit: PropTypes.bool,
+  onEditNetwork: PropTypes.func,
   onRpcUrlAdd: PropTypes.func,
 };
 
```

Comment:
I've put in what looks like valid values but I still cannot submit.  Is this desired?
<img width="361" alt="SCR-20240703-mwst" src="https://github.com/MetaMask/metamask-extension/assets/46655/1720c214-721b-41d2-8791-215eb1635a0b">

Reply:
because the rpc url selected is not a valid one ( we should see an error message here ), let me add some valid RPC to the dummy list for test purpose